portugues/colabora/chora/chora.md

```diff
@@ -1,3 +1,8 @@
 Aos prantos, não lhe resta nada a não ser desistir de usar o app ou conviver com um bug inconveniente.
 Triste vida de alguém com tanto potencial e o desperdiça por falta de coragem...
-A menos que
\ No newline at end of file
+A menos que va ate a bruxa que mora na esquina e peca uma pocao para que os pobres mortais vejam o quao bom voce é.
+Voce chega na casa da Bruxa e 
+
+[Bate na porta, porque nao gosta de apertar campainha](Campainha/Campainha.md)
+[aperta a campainha](Campainha/Campainha.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.